devise_ldap_authenticatable.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency('rake', '>= 0.9')
   s.add_development_dependency('rdoc', '>= 3')
-  s.add_development_dependency('rails', '>= 4.0')
```

Comment:
I think I changed this in the attempt to test something and ended up forgetting to rollback